src/extensions/try-convert/Microsoft.DotNet.UpgradeAssistant.Extensions.TryConvert/TryConvertRunner.cs

```diff
@@ -95,12 +117,16 @@ private async Task<UpgradeStepApplyResult> RunTryConvertAsync(IUpgradeContext co
 
             if (!result)
             {
-                _logger.LogCritical("Conversion with try-convert failed.");
-                return new UpgradeStepApplyResult(UpgradeStepStatus.Failed, $"Conversion with try-convert failed.");
+                var description = "Conversion with try-convert failed.";
+                _logger.LogCritical(description);
```

Comment:
why Critical here vs Debug as was done everywhere else?

Reply:
This is existing code. I have just removed the description out of it. Do you think it should be changed?

Reply:
ah if was already like this, probably fine then.